modules/nixos/common/cgroups.nix

```diff
@@ -1,6 +1,7 @@
+{ config, lib, ... }:
 {
-  nix = {
-    settings = {
+  config = lib.mkIf (lib.hasPrefix "build" config.networking.hostName) {
+    nix.settings = {
       experimental-features = [
         "auto-allocate-uids"
         "cgroups"
```

Comment:
Did you had any more crashes with this?

Reply:
I'd need to reduce max-silent-time to check for crashes.